docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -1,272 +1,124 @@
-. APM Server: {apm-server-ref}/upgrading.html[upgrade instructions]
-
-NOTE: Logstash 5.6 and 6.n and Beats 5.6 and 6.n are compatible with all 6.n versions of
-Elasticsearch. This provides flexibility in when you schedule the upgrades
-for your Logstash instances and Beats agents. We recommend upgrading Logstash
-and Beats as soon as possible to take advantage of performance improvements
-and other enhancements.
-
-=== Upgrading to 6.3
-Starting in 6.3, the default distributions of {es}, {ls}, and {kib}
-include {xpack} and a free Basic license that never expires.
-
-You can perform rolling upgrades to 6.3 from OSS-only clusters running 5.6
-or 6.0-6.2. Basic features are operational once the cluster is fully
-upgraded. If you are already using {xpack}, your settings are preserved when
-you upgrade.
-
-IMPORTANT: If you use {xpack} you must explicitly remove the old {xpack} plugin
-before restarting: `bin/elasticsearch-plugin remove x-pack`. The node will fail
-to start if the {xpack} plugin is present.
-
-You must explicitly enable data collection after the upgrade to use monitoring.
-Set `xpack.monitoring.collection.enabled` to `true` with the `_cluster/settings`
-API:
-
-[source,json]
-----------------------------------------------------------
-PUT /_cluster/settings
-{
-    "persistent" : {
-        "xpack.monitoring.collection.enabled" : "true"
-    }
-}
-----------------------------------------------------------
-// CONSOLE
-
-To take more of the {stack} features for a spin, you can start a 30-day trial
-from Kibana or with the Start Trial API:
-
-[source,json]
-----------------------------------------------------------
-POST _license/start_trial
-----------------------------------------------------------
-// CONSOLE
-
-The 30-day trial enables you to try out the full set of platinum features,
-including security, machine learning, alerting, graph capabilities, and more.
-
-[role="xpack"]
-[[xpack-stack-upgrade]]
-=== Upgrading from 5.6
-
-{xpack} 5.6 provides migration and upgrade APIs for Elasticsearch and a
-Upgrade Assistant UI for Kibana. These tools are included with the trial
-license and the free basic license.
-
-To upgrade to {version} from 5.6:
+. Test the upgrade in a dev environment *before* upgrading your
+production cluster.
 
-. {ref}/setup-upgrade.html[Upgrade Elasticsearch] to the most recent 5.6 and
-install {xpack} on all nodes in your cluster. If you are upgrading from an
-earlier 5.x release, you can perform a rolling upgrade. To upgrade from older
-versions you must perform a full cluster restart.
-+
-If your trial license expires,
-https://register.elastic.co/[register for a free Basic license]. To apply the
-license, upload the license file with the `license` API:
-+
-[source,json]
-----------------------------------------------------------
-license -d @license.json
-----------------------------------------------------------
+. Back up your data. You **cannot roll back** to an earlier version unless
```

Comment:
I ran into a problem when I included the .kibana index in my snapshot. When I subsequently tried to restore this index it conflicted with the existing one. Can someone from @elastic/kibana-platform tell me if this was user error on my part or if it is expected?

If expected, then we should update the docs with guidance on avoiding this pitfall.

Reply:
This feels like general snapshot/restore behavior--you can't restore on top of an open index. As far as the upgrade process goes, we tell folks to back up their data, but don't really get into how to use the snapshot if the upgrade goes sideways. (I made the implicit assumption that they would understand how to use the backup if they needed to.) It might make sense to add a few words when we add the info about what to do if you have a "broken upgrade".

Reply:
> This feels like general snapshot/restore behavior--you can't restore on top of an open index. 

Thanks Deb! That makes sense.

> It might make sense to add a few words when we add the info about what to do if you have a "broken upgrade".

Yes, that would be very helpful! Including a note about what this means for Kibana would also be helpful, I think. Something along the lines of, "Note for Kibana users: If you're restoring a snapshot that contains the .kibana index, you'll need to shut Kibana down, close the index, and restore using the command line".